src/BTR3.ohm

```diff
@@ -1,19 +1,19 @@
 BTR3 {
-	BTRSfile = FileHeader Bank* FileTrailer
+    BTRSfile = FileHeader Bank* FileTrailer
 
     Bank = BankHeader Account* BankTrailer
 
     Account = AccountHeader TransactionDetail* AccountTrailer
 
     
     FileHeader = "01" delim senderIdentification delim receiverIdentification delim fileCreationDate delim fileCreationTime delim fileIdentificationNumber delim physicalRecordLength delim blockSize delim versionNumber eor
-    senderIdentification = alnum +
-    receiverIdentification = alnum +
+    senderIdentification = alphaNumeric+
+    receiverIdentification = alphaNumeric+
```

Comment:
These alterations are hardening the definition, these values cannot include `,` or `/`